### DISCUSSION TITLE
[FeatureRequest] Show units automatically in the multi-axes of the new Timeseries panel

### DISCUSSION THREAD

**Discussion by laurafernandezdiaz** [grafana/grafana#38238] [Dashboards, Panels & Visualizations]
I am using the new Grafana 7.4 Timeseries panel. I can put multiple multi-axes with the units on each one. However, I cannot indicate the unit automatically on the graph. 

For example, if I have the unit stored in the database (influx 2.0), how can I automatically indicate the unit of each variable on each axis? Would it be possible to add an option in that panel that allows adding the query or variable that contains the units of the database? Or would there be another way to automate it?

![image](https://user-images.githubusercontent.com/51406942/110658096-ad3c1800-81c1-11eb-93c3-41ab5f2e0455.png)

**Comment by HogeBlekker** [comment]
This would be a very useful feature indeed.

A typical use case is IIoT in the process industry, where people want to trend e.g. 5 different metrics (temperature, pressure, flow rate, motor RPM and mixing speed, for example), each with their own unit. Rather than having the user set this each time they select a metric, it would be useful if this is preset.

Store this metadata in the timeseries database (e.g. as InfluxDB tag) or a separate datasource (e.g. a relational database mapping each metric name to a unit)?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by laurafernandezdiaz** [comment]
In Grafana 8.1.X a beta functionality called "Config from query results transform" has been added. With this option it is possible to change some settings of the dashboard in Grafana using queries from the database. Therefore, this has been applied to being able to read the unit of the variables from the database so that the graphs are updated directly with it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
